RadixWallet/Features/ImportMnemonic/ImportMnemonic+View.swift

```diff
@@ -95,6 +95,18 @@ extension ImportMnemonic {
 
 		public init(store: StoreOf<ImportMnemonic>) {
 			self.store = store
+			let blue = UIColor(Color.app.blue2)
+			let white = UIColor(Color.app.white)
+			let appearance = UISegmentedControl.appearance()
+			appearance.selectedSegmentTintColor = blue
+
+			// NORMAL
+			appearance.setTitleTextAttributes([.font: FontConvertible.Font.app.segmentedControlNormal], for: .normal)
```

Comment:
second time we use `segmented` control, we can create a wrapping view for it perhaps?

Reply:
Or just an extension on SegmentedControl that sets these values

Reply:
Perhaps it can even be rolled into a picker style

Reply:
fixed in https://github.com/radixdlt/babylon-wallet-ios/pull/935